### ISSUE TITLE
[Markup and meta data] HTML markup isn't valid. (09.01.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#35293]
#### VFS team name
Public Websites

#### VFS product name
Outreach & Events Enhancements

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

---

### Platform Issue
HTML markup isn't valid.

### Issue Details
When filtering by a custom date range, the Start Date and End Date labels are not associated with an input.

### VA.gov Experience Standard
[Category Number 09, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A

---

### Platform Recommendation
Use the <fieldset> / <legend> pattern instead. See https://design.va.gov/storybook/?path=/docs/components-date--default for a similar implementation.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.
